README.md

```diff
@@ -1 +1,20 @@
-# http-observatory-dashboard
+# HTTP Observatory Dashboard
+
+This dashboard provides a metrics dashboard to see the status of the [Observatory](https://observatory.mozilla.org/) for each link.
+
+
+## Install
+
+```
+pip install -r requirements.txt
+```
+
```

Comment:
Should mention that you'd want to modify the sites.json in httpobsdashboard/conf.
Suggestion

```md
## Install and Setup
* Clone this repo
* `pip install -r requirements.txt`
* Add the websites you want to check to the JSON files in httpobsdashboard/conf/
```